.github/workflows/ci.yml

```diff
@@ -14,10 +14,13 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.nodejs }}
+        
+    - name: Update npm
+      run: npm install -g npm@latest
```

Comment:
Why is this necessary?

Reply:
I've upgraded the dependencies and the lockfile is now in v2 (since npm@7), which npm included with old node environments don't support.